src/PhpSpec/Console/ContainerAssembler.php

```diff
@@ -389,7 +389,7 @@ private function setupLocator(IndexedServiceContainer $container)
 
                 $config = array_merge($defaults, $suite);
 
-                if (!is_dir($config['src_path'])) {
+                if (!empty($config['src_path']) && !is_dir($config['src_path'])) {
```

Comment:
I don't like the use of `empty` here (we know it's set from the defaults) - negation alone should be enough I think